src/components/Overview/OverviewCardBody/OverviewCardBody.tsx

```diff
@@ -138,12 +138,11 @@ export type OverviewCardBodyProps = {
   config: ComputedServerConfig;
 };
 
-//{this.state.displayMode === OverviewDisplayMode.EXPAND && this.renderCharts(ns)}
-
 export const OverviewCardBody = (props: OverviewCardBodyProps) => {
+  const display = props.displayMode ? props.displayMode : OverviewDisplayMode.EXPAND;
```

Comment:
Shorter statement using nullish coalescing operator: 
```
const display = props.displayMode ?? OverviewDisplayMode.EXPAND;
```

Reply:
Yeah I was so fast I use to set 
```
const display = props.displayMode || OverviewDisplayMode.EXPAND;
```

Reply:
It is better to use ?? operator instead of ||, it could be dangerous in case of falsy values. (https://stackoverflow.com/questions/61480993/when-should-i-use-nullish-coalescing-vs-logical-or)